plugins/yivi-web/dom-manipulations.ts

```diff
@@ -28,7 +38,7 @@ module.exports = class DOMManipulations {
     if (state.isFinal) {
       this._detachEventHandlers();
       // Make sure all restart buttons are hidden when being in a final state
-      this._element.querySelectorAll('.yivi-web-restart-button').forEach((e) => (e.style.display = 'none'));
+      this._element.querySelectorAll('.yivi-web-restart-button').forEach((e) => ((e as HTMLButtonElement).style.display = 'none'));
```

Comment:
The `querySelectorAll` method is generic, so we can specify the type of the elements like so:

```suggestion
      this._element.querySelectorAll<HTMLButtonElement>('.yivi-web-restart-button').forEach((e) => (e.style.display = 'none'));
```